### ISSUE TITLE
Feedback: "Navigate to the next column in a grid" (Minimal Data Grid, Test 4, Command "'y' (single quick key nav on)", V26.05.21)

### ISSUE THREAD

**Issue by tactics2** [w3c/aria-at#1373]
## Description of Behavior

y is not an available command in voiceover. This should be removed from the test plan.

## Test Setup

- Test File: [test-20-navToNextColumnGrid-voiceover_macos.collected.html](https://aria-at.netlify.app/tests/apg/minimal-data-grid/test-20-navToNextColumnGrid-voiceover_macos.collected.html)
- AT: VoiceOver for macOS (Version 15.6.1)
- Browser: Safari (Version 26)
- Command: `'y' (single quick key nav on)`

**Comment by mcking65** [comment]
Resolved by #1374
